src/hearings/converters/additional-facilities.answer.converter.ts

```diff
@@ -17,11 +17,12 @@ export class AdditionalFacilitiesAnswerConverter implements AnswerConverter {
   public transformAnswer(hearingState$: Observable<State>): Observable<string> {
     return hearingState$.pipe(
       map(state => {
+        let result = '<ul>';
         const facilities = this.route.snapshot.data.additionFacilitiesOptions;
-        const selection = state.hearingRequest.hearingRequestMainModel.hearingDetails.facilitiesRequired
-          .map((facility: string) => AdditionalFacilitiesAnswerConverter.getFacilityValue(facilities, facility));
-
-        return selection.join(', ');
+        state.hearingRequest.hearingRequestMainModel.hearingDetails.facilitiesRequired
+        .forEach((facility: string) => result += `<li>${AdditionalFacilitiesAnswerConverter.getFacilityValue(facilities, facility)}</li>`);
```

Comment:
rather than a loop over the array to do string concat, could you not use `.join('</li><li>')` or something?

Reply:
So I would add the `.map` back, but then return ``<ul><li>{{selection.join('</li><li>')</ul>``

Reply:
I have had before the map way of doing @paddybasi but due to some issue sonar cloud suggestions I have to revert back to use foreach